### PR TITLE
Move schemafiles into web-build-tools

### PR DESCRIPTION
**Pull request by nickpape-msft**

### CODE REVIEW

File: gulp-core-build-typescript/src/ApiExtractorTask.ts [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class ApiExtractorTask extends GulpTask<IApiExtractorTaskConfig>  {
   };
 
   public loadSchema(): Object {
-    return require('./api-extractor.schema.json');
+    return require('./schemas/api-extractor.schema.json');
```
Comment (iclanton) [inline gulp-core-build-typescript/src/ApiExtractorTask.ts]: Should all of the schemas go into a `schemas` directory?

File: test-web-library-build/src/test.scss.ts [hunk 2]
```diff
@@ -2,10 +2,10 @@
 import { loadStyles } from '@microsoft/load-themed-styles';
```
Comment (iclanton) [inline test-web-library-build/src/test.scss.ts]: This file should probably be deleted.

File: gulp-core-build-typescript/src/ApiExtractorTask.ts (hunk 1 above)
Comment (nickpape-msft) [inline gulp-core-build-typescript/src/ApiExtractorTask.ts]: Possibly. I only moved them into a folder once there were multiple schemas

File: test-web-library-build/src/test.scss.ts (hunk 2 above)
Comment (nickpape-msft) [inline test-web-library-build/src/test.scss.ts]: Will do in a second PR here in a sec

Comment (nickpape-msft) [inline test-web-library-build/src/test.scss.ts]: https://github.com/Microsoft/web-build-tools/pull/76

**Pull request merged**